frontend/src/components/contexts/contexts.ts

```diff
@@ -1,14 +1,20 @@
 import * as React from "react";
-import { MockProfileBackendService } from 
-  "../../services/mock_profile_backend/mock_profile_backend";
-import { ProfileHandlerService } from 
+import { MockProfileBackendService } from
+  "../../services/mock_backend/mock_profile_backend";
```

Comment:
nit: I'm not sure about this pattern of splitting imports between multiple lines. Is that personal preference or something the linter is doing?

Reply:
I've held off on using the linter because early on you said it was configured in a way that broke the Google style guide. That said, I've broken them into multiple lines to adhere to the column limit, but could easily put all imports on one line if you think that is preferable to multi-line imports.

Reply:
Ohh, gotcha. Yes, imports are exempt from the line length limit. And I don't think the linter does anything _wrong_, it just doesn't catch everything it could. I looked into configuring it better but it's just missing a bunch of the features our internal linter has.

Reply:
Gotcha, in that case I'll start using it! And thank you for the clarification!